### PR TITLE
Create data.gouv.fr.xml

### PR DESCRIPTION
**Pull request by nicolasleger**

### CODE REVIEW

**Comment by Bisaloo** [comment]
Tests fail on this one because you have rules for subdomains you didn't write a target for beforehand. In this case, the subdomains are:
- `data.gouv.fr`
- `id.data.gouv.fr`

**Comment by nicolasleger** [comment]
@Bisaloo Thanks for your help, I fixed it and regroup my commits in one.

File: src/chrome/content/rules/data.gouv.fr.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<ruleset name="data.gouv.fr">
+        <target host="data.gouv.fr" />
+        <target host="www.data.gouv.fr" />
+        <target host="adresse.data.gouv.fr" />
+        <target host="id.data.gouv.fr" />
+        <target host="inspire.data.gouv.fr" />
```
Comment (J0WI) [inline src/chrome/content/rules/data.gouv.fr.xml]: See also:
agd.data.gouv.fr
cada.data.gouv.fr
inspire.data.gouv.fr
static.data.gouv.fr
territoire.data.gouv.fr
webmail.data.gouv.fr

File: src/chrome/content/rules/data.gouv.fr.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+        <rule from="^http://data\.gouv\.fr/"
+                to="https://www.data.gouv.fr/" />
+        <rule from="^http://id\.data\.gouv\.fr/"
+                to="https://www.data.gouv.fr/" />
```
Comment (J0WI) [inline src/chrome/content/rules/data.gouv.fr.xml]: Please add a comment to document why those redirects are required.

**Comment by nicolasleger** [comment]
@J0WI sorry I don't have the time yet to re-enter in this, feel free to edit it.

**Comment by J0WI** [comment]
Done. Maybe @jeremyn or @gloomy-ghost can review my changes.

**Comment by jeremyn** [comment]
Sublist3r shows several other domains that aren't included such as https://wiki.data.gouv.fr and https://yak.data.gouv.fr. Do you want to add those (not just those two)?

**Comment by J0WI** [comment]
I marked it as `(partial)`.

**Comment by J0WI** [comment]
@jeremyn can we finish this?

**Comment by jeremyn** [comment]
- [ ] https://static.data.gouv.fr gets a `403` so it needs a `test`. There are several hits on Google for this domain.

**Comment by J0WI** [comment]
Okay, I added one.

**Pull request merged**